scripts/script-tag.js

```diff
@@ -57,31 +58,46 @@
         });
     }
 
-    if (typeof moment === 'undefined') {
-        loadScript('https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js', function() {
-            console.log('moment js loaded')
+    function isItemAvailableForOrder (virtualId, variantId) {
+        return new Promise(function(resolve, reject) {
+            jQueryPreOrder.ajax({
+                type: 'GET',
+                // need to update this endpoint to use correct endpoint for checking the product preorder availability
+                url: `${shopUrl}/admin/products/${virtualId}.json`,
+                crossDomain: true,
```

Comment:
Is this needed?

Reply:
I have added it to avoid any issue related to CORS as we will not host our script on Shopify